ideascube/search/migrations/0001_initial.py

```diff
@@ -6,9 +6,10 @@
 
 
 class CreateSearchModel(migrations.CreateModel):
-    def database_forwards(self, *_):
-        # Don't run the parent method, we create the table our own way
```

Comment:
I wish you had kept the comment here, the goal was to explicitly state that we don't want anything `super()`.